### PR TITLE
Update Documentation to include Error Id in response

### PR DESCRIPTION
**Pull request by alicedaydream**
Update the List Item Bulk V4 Documentation to error id in response.

### CODE REVIEW

**Comment by RoAnnC1** [comment]
@alicedaydream 
Hi you'll need to sign the CLA before this can be merged.
Also, there is no need to add additional spacing in the table formatting. The column widths are set automatically by the CSS and build process, you can't adjust them in the markdown.

File: src/api-reference/common/list-item/v4.list-item-bulk.markdown [hunk 1]
```diff
@@ -600,10 +603,10 @@ cache-control: no-cache, private
 
 ### <a name="schema-bulk-list-item-error"></a>Bulk List Item Error
 
-|Name|Type|Format|Description|
-|---|---|---|---|
-|`message`|`string`|-|The detailed error message.|
-|`listItem`|[`BulkListItemRequest`](#schema-bulk-list-item-request)|-|List item corresponding to the error.|
+| Name       |Type|Format|Description|
+|------------|---|---|---|
```
Comment (melissa-von-wald) [inline src/api-reference/common/list-item/v4.list-item-bulk.markdown:607]: `id` needs to be added to this table

Comment (alicedaydream) [reply src/api-reference/common/list-item/v4.list-item-bulk.markdown]: Done!

**Comment by melissa-von-wald** [comment]
Can you also update references of us.api.concursolutions.com to us2.api.concursolutions.com?

**Review by melissa-von-wald — APPROVED** [review]

**Comment by alicedaydream** [comment]
Hi @RoAnnC1,
I've completed the CLA however it looks like the check is still showing as "not signed." Is there anything else that I need to do on my end to get this PR merged?

**Comment by alicedaydream** [comment]
> @alicedaydream I'm not sure what's going on. Do you have more than 1 GitHub account? Didn't you have this issue the last time you did a PR? It looks like you need to link your email to your account based on this message: Alice Day seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> 
> If you need help the #ask-github Slack channel would be your best option. I'm not really a GitHub expert.

Hi @RoAnnC1,
Can you please check again to see if the CLA is now signed and if the PR can be merged? Looks like I needed to set the below values for my "alicedaydream" Github account.
```
git config user.name "Full Name"
git config user.email "email@address.com"
```

**Comment by RoAnnC1** [comment]
@alicedaydream 
Now it says that one of your accounts has signed, but not the other. If you scroll up to the first CLA notice it shows that Alice Day still needs to sign. I'm not sure why that's showing up since it doesn't look like that account is involved in this PR.

**Review by RoAnnC1 — APPROVED** [review]
:shipit:

**Pull request merged**